driver-mapping/src/main/java/com/datastax/driver/mapping/Mapper.java

```diff
@@ -1017,11 +1083,7 @@ boolean isIncludedInQuery() {
             Preconditions.checkNotNull(columnMappers);
             this.queryType = queryType;
             this.columns = columnMappers;
-            this.optionTypes = EnumSet.noneOf(Option.Type.class);
-            for (Option opt : options.values()) {
-                if (opt.isIncludedInQuery())
-                    this.optionTypes.add(opt.type);
-            }
+            this.options = options.values();
```

Comment:
For the record, we want to keep testing `isIncludedInQuery()`, to avoid storing extra prepared statement when unnecessary. For example, ttl(100) and ttl(2000) both generate the same query containing "TTL (?)", so it's not efficient to reprepare the query for each one.